src/mono/wasm/runtime/debug.ts

```diff
@@ -59,6 +61,7 @@ export function mono_wasm_add_dbg_command_received(res_ok: boolean, id: number,
 }
 
 function mono_wasm_malloc_and_set_debug_buffer(command_parameters: string) {
+    const heapU8 = localHeapViewU8();
```

Comment:
this is incorrect because we're calling malloc after, we need to do this after the malloc call